apps/portals/ampals/src/config/scripts/exportS3ProductionBucketName.sh

```diff
@@ -0,0 +1,2 @@
+export S3_PRODUCTION_BUCK_LOCATION=s3://prod.ampals.synapse.org
+export CF_PRODUCTION_DIST_ID=????????
```

Comment:
Hope this doesn't break things :) Has this been set up?

Reply:
Chicken and the egg problem.  I think Xa is waiting for this configuration file to update to the correct value (and set up the dns)

Reply:
prod: E1J1L5FUOCXZHU
staging: E27YTHLWH817NN

Reply:
Wait... Was it not amp-als.org?